frontend/src/metabase/visualizations/components/TableInteractive.jsx

```diff
@@ -345,12 +345,15 @@ export default class TableInteractive extends Component {
           transition: dragColIndex != null ? "left 200ms" : null,
           backgroundColor,
         }}
-        className={cx("TableInteractive-cellWrapper", {
+        className={cx("TableInteractive-cellWrapper text-dark", {
           "TableInteractive-cellWrapper--firstColumn": columnIndex === 0,
           "TableInteractive-cellWrapper--lastColumn":
             columnIndex === cols.length - 1,
+          "TableInteractive-emptyCell": value == null,
           "cursor-pointer": isClickable,
           "justify-end": isColumnRightAligned(column),
+          "TableInteractive--PK": isID(column),
```

Comment:
Should this be `"TableInteractive--PK": isPK(column),`?

Reply:
Or `"TableInteractive--ID": isID(column),`

Reply:
Yeah was just about to suggest the `TableInteractive--ID` solution 👍